astroquery/fermi/core.py

```diff
@@ -152,9 +151,9 @@ def __call__(self, result_url, check_frequency=1, verbose=False):
         return fitsfile_urls
 
     def _check_page(self):
-        result_page = commons.send_request(self.result_url,
-                                           None,
-                                           self.TIMEOUT)
+        result_page = requests.post(url=self.result_url,
```

Comment:
No, it doesn't really match the layout of the `BaseQuery` class, as you said, but we still ought to be using some of the `BaseQuery` features here.  Again, though, it's meant for a different issue.